doc/tutorials/introduction_to_MNE.rst

```diff
@@ -0,0 +1,105 @@
+
+**figure/diagram plot**
+
+
+What can you do with MNE Python?
+--------------------------------
+
+   - **Raw data visualization** to visualize recordings, can also use
+     *mne_browse_raw* for extended functionality (see :ref:`ch_browse`)
+   - **Epoching**: Define epochs, baseline correction, handle conditions etc.
+   - **Averaging** to get Evoked data
+   - **Compute SSP pojectors** to remove ECG and EOG artifacts
+   - **Compute ICA** to remove artifacts or select latent sources.
+   - **Maxwell filtering** to remove environmental noise.
+   - **Boundary Element Modeling**: single and three-layer BEM model
+     creation and solution computation.
+   - **Forward modeling**: BEM computation and mesh creation
+     (see :ref:`ch_forward`)
+   - **Linear inverse solvers** (dSPM, sLORETA, MNE, LCMV, DICS)
+   - **Sparse inverse solvers** (L1/L2 mixed norm MxNE, Gamma Map,
+     Time-Frequency MxNE)
+   - **Connectivity estimation** in sensor and source space
+   - **Visualization of sensor and source space data**
+   - **Time-frequency** analysis with Morlet wavelets (induced power,
+     intertrial coherence, phase lock value) also in the source space
+   - **Spectrum estimation** using multi-taper method
+   - **Mixed Source Models** combining cortical and subcortical structures
+   - **Dipole Fitting**
+   - **Decoding** multivariate pattern analyis of M/EEG topographies
+   - **Compute contrasts** between conditions, between sensors, across
+     subjects etc.
+   - **Non-parametric statistics** in time, space and frequency
+     (including cluster-level)
+   - **Scripting** (batch and parallel computing)
+
+
+Is that all you can do with MNE-Python?
+---------------------------------------
+No!
+
+    - detect heart beat QRS component
+    - detect eye blinks and EOG artifacts
+    - compute SSP projections to remove ECG or EOG artifacts
+    - compute Independent Component Analysis (ICA) to remove artifacts or
+      select latent sources
+    - estimate noise covariance matrix from Raw and Epochs
+    - visualize cross-trial response dynamics using epochs images
+    - compute forward solutions
+    - estimate power in the source space
+    - estimate connectivity in sensor and source space
+    - morph stc from one brain to another for group studies
+    - compute mass univariate statistics base on custom contrasts
+    - visualize source estimates
+    - export raw, epochs, and evoked data to other python data analysis
+      libraries e.g. pandas
+    - and many more things ...
+
+
+What you're not supposed to do with MNE-Python
+----------------------------------------------
+
+    - **Brain and head surface segmentation** for use with BEM
+      models -- use Freesurfer.
+    - **Raw movement compensation** -- use Elekta Maxfilter™
```

Comment:
This actually isn't true anymore, we do have raw movement comensation as of a month or two ago, and I forgot to change it. It should now probably say:
```
- **Continuous head position estimation** -- use Elekta MaxFilter
```
because althogh we have the algorithm, we haven't vetted it. And the `Raw movement compensation` line can be removed entirely, because it's actually part of Maxwell filtering.